tests/check_doctest_names.py

```diff
@@ -164,6 +164,9 @@ def check_doctests(modules=None, respect_skips=True, verbose=True,
     fails = len(failures)
     passes = total - fails
     print(f'\n{passes} passes and {fails} failures out of {total} total doctests.\n')
+    if not fails:
+        return
+
     print('List of failures:')
```

Comment:
If we raise an exception here, we can then report this as an error in our CI.

Reply:
I've changed execution to call `sys.exit(1)` when failures are found, I think this is appropriate for CI.